src/index.js

```diff
@@ -0,0 +1,70 @@
+import Schedule from './module/classSchedule.js';
+import Storage from './module/classStorage.js';
+import Task from './module/classTask.js';
+import './style.css';
+import imgplus from './images/plus.svg';
+import imgreload from './images/reload.svg';
+import imgpencil from './images/pencil.svg';
+
+const myPencil = new Image();
+myPencil.src = imgpencil;
+myPencil.className = 'stop';
+
+const reloadDiv = document.querySelector('.reload');
+const myReload = new Image();
+myReload.src = imgreload;
+reloadDiv.appendChild(myReload);
+
+const plusDiv = document.querySelector('.plus');
+const myPlus = new Image();
+myPlus.src = imgplus;
+plusDiv.appendChild(myPlus);
+
+// event display tasks
+
+document.addEventListener('DOMContentLoaded', Schedule.displayTasks);
+
+// event add a task
+
+myPlus.addEventListener('click', () => {
+  const tasktext = document.querySelector('#task-value').value; // get value
+  const code = Storage.getTasks().length + 1;
+  const task = new Task(code, tasktext, false); // create the object
+  Schedule.addTaskToList(task); // add task to the screen list
+  Storage.addTask(task); // Add task to the storage
+  Schedule.clearInput();
```

Comment:
- [ ] [OPTIONAL] Kindly make sure not to use too many unnecessary comments instead you can give your functions, and variables descriptive names